### ISSUE TITLE
Nukie Rounds should end immediately when all nukies are dead

### ISSUE THREAD

**Issue by keronshb** [space-wizards/space-station-14#21617]
## Description

Nobody wants to wait a long time after the round ends. It's just bad gameplay design if players are forced to wait after all the destruction.

The only exception to this is that it will need to be revisited when Dynamic becomes a thing.

**Comment by asperger-sind** [comment]
certified reimplentation moment

**Comment by daerSeebaer** [comment]
So like the opposite of what was merged in #19850 ?

**Comment by Emisse** [comment]
yea that kinda thing probably shoulda waited for dynamic but i dont think its a huge issue

**Comment by GoodWheatley** [comment]
As someone who’s played with both the round ending instantly and the shuttle call, I can say that the round instantly ending is super lame and doesn’t allow for any closure or RP. If ghosts having to sit around is a massive issue, reducing the shuttle arrival time to something like three minutes is a good compromise.

**Comment by PokkeTigrex** [comment]
> As someone who’s played with both the round ending instantly and the shuttle call, I can say that the round instantly ending is super lame and doesn’t allow for any closure or RP. If ghosts having to sit around is a massive issue, reducing the shuttle arrival time to something like three minutes is a good compromise.

If anything, keep the shuttle call just for MRP+ for the time being, LRP may be better for a instant round over.

**Comment by LordEclipse** [comment]
I agree that the shuttle should just come quicker. Like 3 minutes as was previously said. I think we should avoid ending the round instantly as it just doesn't feel proper. And maintaining consistency is better than anything else in terms of rounds ending.

**Comment by IProduceWidgets** [comment]
Alternatively, if someone wanted to do the automated ERT thing this would also be a great first place to try some engi/medical ERT. That would add ghost roles for the dead and give them something to do for 10 minutes.

**Comment by BYONDFuckery** [comment]
I might decrease the shuttle time even further to 1 minute

**Comment by Brandon-Huu** [comment]
Is this something you guys actually want?

**Comment by PolterTzi** [comment]
> > As someone who’s played with both the round ending instantly and the shuttle call, I can say that the round instantly ending is super lame and doesn’t allow for any closure or RP. If ghosts having to sit around is a massive issue, reducing the shuttle arrival time to something like three minutes is a good compromise.
> 
> If anything, keep the shuttle call just for MRP+ for the time being, LRP may be better for a instant round over.

I feel like even LRP would benefit from some time of after-conflict play to unwind, celebrate and whatnot.

Instantly ending the round after one team is dead feels like a more pvp action game move than the "simulation"/RP with action elements vibe the game seems to be aiming for.
Even in LRP, being a pvp arena is not really the goal and people can still want to RP a bit, goof off or struggle to recover from the aftermath.

**Comment by VoidMeticulous** [comment]
Should this be updated? The issue was raised almost two years ago with mixed reception. @PolterTzi makes a great point in that immediate ending of the round might not be universally desired, which is corroborated by the reactions to the issue itself.

**Comment by juicyslew** [comment]
Seems like this issue should be closed?
